components/geometry-merger/examples/basic/index.html

```diff
@@ -4,33 +4,27 @@
     <meta charset="utf-8">
     <title>A-Frame Geometry Merger Component - Basic</title>
     <meta name="description" content="Basic example for Geometry Merger component."></meta>
-    <script src="https://aframe.io/releases/0.7.1/aframe.min.js"></script>
-    <script src="../../dist/aframe-geometry-merger-component.min.js"></script>
-    <script src="https://unpkg.com/aframe-environment-component@1.0.0/dist/aframe-environment-component.min.js"></script>
+    <script src="https://aframe.io/releases/1.4.1/aframe.min.js"></script>
```

Comment:
Can be deleted once we have a release that includes #5250 & we update to it.  For now this is needed.